walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,90 @@
 
 package com.google.sps;
 
+import java.util.Queue;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Comparator;
+import java.util.ArrayList;
+import java.util.List;
+
+
+// A meeting request has:
```

Comment:
For formatting comments like these, it's good to use javadoc style.

https://github.com/google/google-java-format provides a way to automatically format these (I think if you upload the jar to your cloud shell instance, you can use it there)